README.md

```diff
@@ -34,10 +34,17 @@ The languages provided here can be used for *making scripts through the GDNative
 1. **[Nim](https://github.com/pragmagic/godot-nim) 👥🔌**
 1. **[JavaScript](https://github.com/GodotExplorer/ECMAScript) 👥🔌**
 1. **[C#](https://docs.godotengine.org/en/stable/getting_started/scripting/c_sharp/index.html) 💍🔌**
-
+1. **[Lua](https://github.com/perbone/luascript) 👥🧬🔌**
```

Comment:
@willnationsdev I overlooked this: the two icons shouldn't be used together, people could get confused about whether they exclusively need an external editor for Lua or not.

Reply:
What about changing the icons to say, "Supports Integrated Editor" and "Supports External Editor"? It can then be implied based on the absence of the integrated one whether it's required to use the external one or not. Because, the Lua documentation specifies that you can use either one (though I haven't checked the specifics in practice).

Reply:
@willnationsdev Indeed, I've misworded the former - I wanted to imply exactly what you meant.

Reply:
@perbone what's your take on this?

Reply:
The "supports" wording seems less confusing indeed.